package.json

```diff
@@ -12,6 +12,7 @@
     "i18next-xhr-backend": "1.5.1",
     "immutability-helper": "2.5.1",
     "moment": "2.22.1",
+    "node-sass": "^4.11.0",
```

Comment:
this and react-svg are the only ones we are using the latest minor version. Is it relatively safe to do that?

Reply:
Should be. No vulnerabilities reported by npm-audit. App seems to work.